packages/split-button/src/SplitButton/SplitButton.types.ts

```diff
@@ -135,6 +135,11 @@ export interface InternalSplitButtonProps
    * The text that will appear inside of the primary button.
    */
   label: string;
+
+  /**
+   * A test id put onto the root element.
+   */
+  'data-testid'?: string;
```

Comment:
We have a helper for this in `lib` -- you could extend `LgIdProps`.

Reply:
`LgIdProps` doesn't have `data-testid`? https://github.com/mongodb/leafygreen-ui/blob/94e895ae758e11b1fb15296b7cf31df66f7270c4/packages/lib/src/LgIdProps/index.ts#L10